frontend/src/Components/EventsCard.jsx

```diff
@@ -17,9 +17,10 @@ const EventsCard = (props) => {
     useEffect(getVolunteersList, [reload]);
 
     useEffect(() => {
-        const checkIfVolunteerSignedForEvent = () => {
+        const checkIfVolunteerSignedForEvent = async () => {
             let found = false;
-            let accepted = false;
+            let accepted = '';
```

Comment:
Same thing here, for constancy it's better to have `let accepted = false`

Reply:
I fixed it back.